IMPROVEMENT better code maintainablity by removing duplicated code (#2)
Nilstrieb/bool-toggle: README.md

It is only available when compiling with `--cfg enterprise_license` and obtaining an enterprise license.
For license inquiries, send mail to `/dev/null`.

## Architecture

> Tis Crate features the world's most resilient bit filping algorithm. It's code adheres to the highest clean code standards
> and aims to provide a maintainable and future proof solution to all the boolean toggling needs there are.

Reviewer: ```suggestion
This Crate features the world's most resilient bit flipping algorithm. Its code adheres to the highest clean code standards
and aims to provide a maintainable and future proof solution to all the boolean toggling needs there are.
```
